blivet/partitioning.py

```diff
@@ -606,9 +606,7 @@ def do_partitioning(storage, boot_disk=None):
     try:
         allocate_partitions(storage, disks, partitions, free, boot_disk=boot_disk)
         grow_partitions(disks, partitions, free, size_sets=storage.size_sets)
-    except Exception:
-        raise
-    else:
+
```

Comment:
Hmm, I prefer the original form. The intention is clearer. Can we just do a whitelist/disable comment instead?

Reply:
Yes, it is possible to disable the warning, fixed.